data/tlg0012/tlg001/tlg0012.tlg001.perseus-grc2.xml

```diff
@@ -32,12 +34,20 @@
                 <biblStruct>
                     <monogr>
                         <author>Homer</author>
-                        <title>Iliadis (Iliad) in two volumes. David B. Monro and Thomas W. Allen, eds. Editio Tertia (Third Edition).</title>
+                        <title xml:lang="lat">Homeri Opera</title>
+                        <editor>David B. Monro</editor>
+                        <editor>Thomas W. Allen</editor>
+                        <edition xml:lang="lat">Editio Tertia</edition>
                         <imprint>
-                            <publisher>Oxford, Oxford University Press</publisher>
-                            <date>1920</date>
+                            <pubPlace>Oxford</pubPlace>
+                            <publisher xml:lang="lat">E Typographeo Clarendoniano</publisher>
```

Comment:
Clarendon Press

Reply:
I've changed this and will update the pull request! On to the Odyssey!

Reply:
Please add the "Press": I think we use that elsewhere. I can't add to this file online.

Reply:
Oops, I will do that! Sorry!